server/repository/src/migrations/v2_01_00/program.rs

```diff
@@ -0,0 +1,39 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+            ALTER TABLE program ALTER COLUMN master_list_id DROP NOT NULL;
+            ALTER TABLE program ADD COLUMN IF NOT EXISTS is_immunisation BOOLEAN NOT NULL DEFAULT false;
+            "#
+        )?;
+    } else {
+        sql!(
+            connection,
+            r#"
+            CREATE TABLE tmp_program (
+                id TEXT NOT NULL PRIMARY KEY,
+                master_list_id TEXT,
+                name TEXT NOT NULL,
+                context_id TEXT NOT NULL,
+                is_immunisation BOOLEAN NOT NULL
+            );
+            INSERT INTO tmp_program SELECT id, master_list_id, name, context_id, false FROM program;
+            DROP TABLE program;
+            CREATE TABLE program (
```

Comment:
I avoided using rename, after the bug we'd seen with SQLite before.
Might be ok now that we're using diesel 2, but didn't want to risk it...